IWXXM/ReleaseNotes-IWXXM.txt

```diff
@@ -1,3 +1,42 @@
+=========================================
+2025-2 Release Candidate 2 - July 2025
+=========================================
+Package SIGMET (Version 4.0.1 - No change)
+  * Corrected all latitudes and longitudes of points in the polygon in sigmet-multi-location-VA.xml
+Package Volcanic Ash Advisory (Version 3.2.0RC2)
+  * Renamed iwxxm:sourceElevation to iwxxm:sourceElevationAMSL and make subsequent changes to Schematron rules and examples
+  * Updated example va-advisory-A7-2.xml
+  * Changed feature type of iwxxm:Volcano to metce:Volcano
+Package Space Weather Advisory (Version 3.1.0RC2)
+  * Changed the multiplicity of iwxxm:locationIndicator from [0..1] to [1..6] to cater for having multiple space weather bands
+    and the mandatory need to specify the element
+  * Updated example spacewx-A7-3.xml to show an improved way to encode the geometry of the affected volume in a Space Weather Advisory
+  * Updated all Space Weather Advisory examples to show the correct way to indicate nil information in iwxxm:locationIndicator
+Package WAFS Significant Weather Forecast (Version 1.2.0RC2)
+  * Revised iwxxm:WAFSSignificantWeatherForecast to allow for extension
+  * Revised to use codelists http://codes.wmo.int/iwxxm/MeteorologicalFeature and http://codes.wmo.int/iwxxm/nil
+Package Quantitative Volcanic Ash Concentration Information (Version 1.0.0RC2)
+  * Removed codelist http://codes.wmo.int/iwxxm/VolcanicAshConcentrations. Amended example qvaci-Example.xml accordingly
+  * Introduced iwxxm:remarks for provision of additional information in the message
+  * Revised iwxxm:QuantitativeVolcanicAshConcentrationInformation to allow for extension
+  * Revised to use codelists http://codes.wmo.int/iwxxm/MeteorologicalFeature and http://codes.wmo.int/iwxxm/nil
+Package Volcano Observatory Notice for Aviation (Version 1.0.0RC2)
+  * Replaced Aviation Colour Code enumeration with codelist http://codes.wmo.int/iwxxm/AviationColourCode
+  * Revised example vona-A7-1.xml
+  * Revised iwxxm:VocanoObservatoryNoticeForAviation to allow for extension
+  * Revised to use codelists http://codes.wmo.int/iwxxm/MeteorologicalFeature and http://codes.wmo.int/iwxxm/nil
+Package MeteorologicalFeature (Version 1.1.0RC2)
+  * Revised iwxxm:MeteorologicalFeature to allow for extension
+  * Revised to use codelists http://codes.wmo.int/iwxxm/MeteorologicalFeature and http://codes.wmo.int/iwxxm/nil
```

Comment:
```suggestion
  * Revised to use codelists http://codes.wmo.int/iwxxm/MeteorologicalFeature and http://codes.wmo.int/iwxxm/nil
  * Revised references of MSL to "the vertical reference"
  * added features for volanoes
* ```

Reply:
No need to update.  The remaining diffs are improvements to documentation.  Previous mentioning of "MSL" is too restrictive, and that's why we change it to "vertical reference" which includes MSL.